dev-client/src/screens/ManageTeamMemberScreen.tsx

```diff
@@ -84,42 +86,39 @@ export const ManageTeamMemberScreen = ({
       AppBar={
         <AppBar title={project?.name} LeftButton={<ScreenCloseButton />} />
       }>
-      <VStack mx="14px">
-        <VStack mt="22px" mb="15px">
-          <Text variant="body1" fontWeight={700}>
-            {formatName(user.firstName, user.lastName)}
-          </Text>
-          <Text numberOfLines={1} variant="body1">
-            {user.email}
-          </Text>
-        </VStack>
-        <RadioBlock<ProjectRole>
-          labelProps={{variant: 'secondary'}}
-          label={t('projects.manage_member.project_role')}
-          options={{
-            MANAGER: {
-              text: t('general.role.MANAGER'),
-              helpText: t('projects.manage_member.manager_help'),
-            },
-            CONTRIBUTOR: {
-              text: t('general.role.CONTRIBUTOR'),
-              helpText: t('projects.manage_member.contributor_help'),
-            },
-            VIEWER: {
-              text: t('general.role.VIEWER'),
-              helpText: t('projects.manage_member.viewer_help'),
-            },
-          }}
-          groupProps={{
-            onChange: setSelectedRole,
-            value: selectedRole,
-            name: 'selected-role',
-          }}
-        />
+      <ScreenContentSection title={t('projects.manage_member.title')}>
+        <VStack>
+          <Box ml="md" my="lg">
+            <MinimalUserDisplay user={user} />
```

Comment:
Hmm I couldn't think of a great way to combine them without introducing more complexity than I feel like this solution does. I'd kinda rather have two components with similar guts laid out differently, than one component with lots of conditional logic.

Reply:
But lmk if you have thoughts on how!

Reply:
I think I agree with @knipec 's assessment here - reusing code seems like more trouble than it's worth, the components aren't particularly large, they're fulfilling different roles, and it seems. In the component designs I think I'd want them to be listed as two distinct components rather than one with two operating modes. If we wanted to factor out something common, I think that the `Image` component could become a standard `ProfilePicture` component, since that's pretty self-contained and could reasonably appear in other parts of the application.